lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -84,10 +84,14 @@ MACHINE. For each set of dupes, only the latest job found is included in
 the return array.
 
 =cut
-sub latest_jobs {
-    my ($self, $until) = @_;
+sub latest_jobs ($self, $until = undef) { @{$self->latest_jobs_with_limit($until)->{jobs}} }
```

Comment:
It would be nice to avoid copying the array here but I suppose one cannot have everything.

Reply:
No big difference to how it was before.